src/smokeTest/java/uk/gov/hmcts/reform/jobscheduler/controllers/JobsControllerSmokeTest.java

```diff
@@ -3,33 +3,41 @@
 import io.restassured.RestAssured;
 import org.junit.Before;
 import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.springframework.beans.factory.annotation.Value;
+import org.springframework.core.io.ClassPathResource;
+import org.springframework.core.io.Resource;
+import org.springframework.core.io.support.PropertiesLoaderUtils;
 import org.springframework.http.HttpHeaders;
-import org.springframework.test.context.junit4.SpringRunner;
 
-@RunWith(SpringRunner.class)
-public class JobsControllerSmokeTest {
+import java.io.IOException;
+import java.util.Map;
+import java.util.Properties;
 
-    @Value("${default-test-url:http://localhost:8484}")
-    private String testUrl;
+public class JobsControllerSmokeTest {
 
     @Before
-    public void setup() {
-        System.out.println(testUrl);
-        RestAssured.baseURI = testUrl;
+    public void setup() throws IOException {
+        Map<String, String> env = System.getenv();
```

Comment:
How about just using `System.getProperty("TEST_URL")`?

Reply:
definitely - thanks

Reply:
Sorry I miss-read, no we can't do that as it's not a property, it's an environment variable.